WBooks/LibraryViewController.swift

```diff
@@ -9,34 +9,75 @@
 import UIKit
 
 final class LibraryViewController: UIViewController {
-    
+
     private let libraryView: LibraryView = LibraryView.loadFromNib()!
     
+    let spacingBetweenCells: CGFloat = 10
+    
+    // Hard coded data array
+    private var bookArray: Array = [["title": "When the doves disappeared", "author": "Timothy Cross", "img": "img_book1"], ["title": "When the doves disappearedasdasdasdasdasasdasdas", "author": "Sofi Oksanen", "img": "img_book2"], ["title": "The best book in the world", "author": "Peter Stjerstrom", "img": "img_book3"], ["title": "Be creative", "author": "unknown", "img": "img_book4"], ["title": "Redesign the web", "author": "Wolox", "img": "img_book5"], ["title": "Yellow", "author": "Matias Schwalb", "img": "img_book6"]]
+    
     override func loadView() {
         view = libraryView
     }
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        /*
-        view.backgroundColor = UIColor.white
-        
-        let frame = CGRect(x: 0, y: view.frame.size.height / 2 - 11, width: view.frame.size.width, height: 22)
-        let label = UILabel(frame: frame)
-        label.text = "Welcome! This is a new blank project"
-        label.textAlignment = .center
-        label.sizeToFit()
-        view.addSubview(label)
-        */
+        let nib = UINib.init(nibName: "MyCustomCell", bundle: nil)
```

Comment:
I think that the `init` part can be removed. We can write it like this: `UINib(nibName: "MyCustomCell", bundle: nil)`